### DISCUSSION TITLE
[HELP] Unable to resolve module crypto

### DISCUSSION THREAD

**Discussion by K9Developer** [octokit/octokit.js#2339] [Q&A] [answered]
Hi guys!
I have a problem, I'm trying to install octokit and use it on my react native app...
I install it and it all goes well, but when I run it it says:
```
Error: Unable to resolve module crypto from ... crypto could not be found within the project or in these directories:
```
I installed it on an empty project (NodeJS not react native) and it worked... So I'm not sure what the problem could be.
Is it interfering with other dependencies?

Thanks.

**Comment by K9Developer** [comment, marked as answer]
After I investigated further I found it to be this line: `const octokit = new Octokit({ auth: token });`

**Reply by kfcampbell** [reply]
@KingOfTNT10 is this the same problem as #2340? Perhaps we can mark the discussion as resolved and close the issue.

**Reply by K9Developer** [reply]
> @KingOfTNT10 is this the same problem as #2340? Perhaps we can mark the discussion as resolved and close the issue.

Yes I solved it

**Reply by kfcampbell** [reply]
Great! Would you mind marking the discussion as "Answered"?

**Reply by K9Developer** [reply]
> Great! Would you mind marking the discussion as "Answered"?

How do I do that?
